docs/config/README.md

```diff
@@ -2208,20 +2208,30 @@ To enable it, set `disabled` to `false` in your configuration file.
 
 ### Options
 
-| Option     | Default                     | Description                                            |
-| ---------- | --------------------------- | ------------------------------------------------------ |
-| `format`   | `[$symbol$status]($style) ` | The format of the module                               |
-| `symbol`   | `"✖"`                       | A format string representing the symbol for the status |
-| `style`    | `"bold red"`                | The style for the module.                              |
-| `disabled` | `true`                      | Disables the `status` module.                          |
+| Option                    | Default                     | Description                                            |
+| ------------------------- | --------------------------- | ------------------------------------------------------ |
+| `format`                  | `[$symbol$status]($style) ` | The format of the module                               |
+| `symbol`                  | `"✖"`                       | The symbol displayed on program error                  |
+| `not_executable_symbol`   | `"🚫"`                       | The symbol displayed when file isn't executable        |
+| `not_found_symbol`        | `"🔍"`                       | The symbol displayed when the command can't be found   |
+| `sigint_symbol`           | `"🧱"`                       | The symbol displayed on SIGINT (Ctrl + c)              |
+| `signal_symbol`           | `"⚡"`                       | The symbol displayed on any signal |
+| `style`                   | `"bold red"`                | The style for the module.                              |
+| `map_symbol`              | `false`                     | Enable symbols mapping from exit code                  |
+| `disabled`                | `true`                      | Disables the `status` module.                          |
 
 ### Variables
 
-| Variable | Example | Description                          |
-| -------- | ------- | ------------------------------------ |
-| status   | `127`   | The exit code of the last command    |
-| symbol   |         | Mirrors the value of option `symbol` |
-| style\*  |         | Mirrors the value of option `style`  |
+| Variable                | Example | Description                                                             |
+| ----------------------- | ------- | ----------------------------------------------------------------------- |
+| status                  | `127`   | The exit code of the last command                                       |
+| status_int              | `127`   | The exit code of the last command                                       |
+| status_common_meaning   | `ERROR` | Meaning of the code if not a signal                                     |
```

Comment:
I think `status_common_meaning` and `status_signal_name` should be combined (they are mutually exclusive anyway). Please also add an option to disable recognizing signal exit codes. Chances are high those are false positives on windows and it should help alleviate the need for `status_signal_name` .

Reply:
Indeed, there are mutually exclusive, but one might want to display text for common meanings and signal numbers for signals, combining both would prevent this.

I'll add an option to ignore signal matching